loops.js

```diff
@@ -6,7 +6,9 @@
 // Hoera!
 // ==========================================
 
-
+for (  i=0; i < 3; i++ ) {
```

Comment:
Hier ontbreek het woordje let. Ik zag in je commits dat je in loops-and-arrays.js deze fout al had opgemerkt en verbeterd, dus goed gedaan! Ik vemoed dat je deze pagina bent vergeten. Ook zie ik dat er te weinig spaties in je code zitten bv. **i=0**  kan beter worden geschreven als **i = 0**. Je kunt hier gemakkelijk de toetsencombinatie  **Ctrl + Alt + L** voor gebruiken. Dan past webstorm het automatisch voor je aan.